Makefile

```diff
@@ -63,10 +63,11 @@ test-e2e: ## Run openshift specific e2e test
 	# failures and flakes.
 	# Feature:Operator tests remove deployments. Thus loosing all the logs
 	# previously acquired.
-	hack/ci-integration.sh $(GINKGO_ARGS) -p -focus="Feature:Operators" || (hack/junitmerge.sh && exit 1)
-	hack/ci-integration.sh $(GINKGO_ARGS) -p -skip="Feature:Operators|Autoscaler" || (hack/junitmerge.sh && exit 1)
+	hack/ci-integration.sh $(GINKGO_ARGS) -p -focus="Feature:Operators" -skip="Serial" || (hack/junitmerge.sh && exit 1)
+	hack/ci-integration.sh $(GINKGO_ARGS) -p -skip="Feature:Operators|Autoscaler|Serial" || (hack/junitmerge.sh && exit 1)
 	# TODO: parallelise autoscaler
-	hack/ci-integration.sh $(GINKGO_ARGS) -focus="Autoscaler" || (hack/junitmerge.sh && exit 1)
+	# Run serial tests only
+	hack/ci-integration.sh $(GINKGO_ARGS) -focus="Autoscaler|Serial" || (hack/junitmerge.sh && exit 1)
```

Comment:
One concern is that we could have a test named with `Serial` in it that may not be intended to be serial in this sense, eg `Serialize content of foo`, at the moment, this would match that, should we instead update these to be more explicit around matching with `[]` as well? eg. `--skip="[Serial]"`, this would leave no ambiguity about the intention

Reply:
Was thinking about that, let me check how the regex will react on the change.